Makefile

```diff
@@ -312,7 +312,7 @@ test-failover:
 # executes terra-tests
 .PHONY: terratest
 terratest: # Run terratest suite
-	cd terratest/test/ && go mod download && go test -v
+	cd terratest/test/ && go mod download && go test -v -timeout 12m -parallel=12
```

Comment:
```suggestion
	cd terratest/test/ && go mod download && go test -v -timeout 15m -parallel=12
```

Looks like we need at least 15m for very e2e real cluster scenario in context of #350

Reply:
@ytsarev, sounds reasonable..